.all-contributorsrc

```diff
@@ -37,5 +37,8 @@
       ]
     }
   ],
-  "repoType": "github"
+  "contributorsPerLine": 7,
+  "repoHost": "https://github.com",
+  "repoType": "github",
+  "skipCi": true
```

Comment:
Let's set this to false because otherwise this can meet up automatic releases

Reply:
Only in this repo or also in the template? 🤔